packages/url_launcher/url_launcher_web/example/integration_test/link_widget_test.dart

```diff
@@ -34,6 +34,7 @@ void main() {
       ));
       // Platform view creation happens asynchronously.
       await tester.pumpAndSettle();
+      await tester.pump();
```

Comment:
Talk to @eyebrowsoffire about needing this extra pump here (also @mdebbar)

Reply:
The difference with the code above, is that (I believe) `pumpAndSettle` has a built-in delay, which seems to be enough in the test cases of the link widget.